tools/sgen/input/FluentUIStyle.yml

```diff
@@ -389,3 +389,64 @@ MSFPrimaryHeaderTokens extends MSFHeaderFooterTokens:
   textColor: $Colors.Foreground.neutral1
   textFont: $Typography.headline
   topPadding: $Spacing.medium
+
+AP_MSFNotificationTokens:
+  backgroundColor: $Colors.Brand.tint40
+  foregroundColor: $Colors.Brand.shade10
+  cornerRadius: $Border.radius.xLarge
+  presentationOffSet: $Spacing.small
+  horizontalPadding: 16pt
+  verticalPadding: 12pt
+  verticalPaddingForOneLine: 16pt
+  horizontalSpacing: 16pt
+  minimumHeight: 64.0
+  minimumHeightForOneLine: 56.0
+  shadowColor: $Shadow.shadow16.color1
+  shadowBlur: $Shadow.shadow16.blur1
+  shadowOffsetX: $Shadow.shadow16.x1
+  shadowOffsetY: $Shadow.shadow16.y1
+
+MSFPrimaryToastNotificationTokens extends MSFNotificationTokens:
+  cornerRadius: $Border.radius.xLarge
+  presentationOffSet: $Spacing.small
+  backgroundColor: $Colors.Brand.tint40
```

Comment:
This is already defined in the AP_MSFNotificationTokens. No need to override it here.